packages/kbn-i18n/GUIDELINE.md

```diff
@@ -283,13 +283,19 @@ Splitting sentences into several keys often inadvertently presumes a grammar, a
 
 ### Unit tests
 
-When testing React component that use the injectI18n higher-order component, use the shallowWithIntl helper function defined in test_utils/enzyme_helpers to render the component. This will shallow render the component with Enzyme and inject the necessary context and props to use the intl mock defined in test_utils/mocks/intl.
+Testing React component that use the injectI18n higher-order component is more complicated because `injectI18n()` creates a wrapper component around the component defined in the app.
```

Comment:
__nit:__ maybe `the component defined in the app` ---> `the original component`?